gen-ai-rag-vertex-ai-vector-search/README.md

```diff
@@ -9,8 +9,15 @@ Center](https://cloud.google.com/architecture/).
 To try this sample:
 
   1. [Enable the required APIs](https://console.cloud.google.com/flows/enableapi?apiid=run.googleapis.com,pubsub.googleapis.com,aiplatform.googleapis.com,iam.googleapis.com,storage.googleapis.com,cloudfunctions.googleapis.com,eventarc.googleapis.com,cloudbuild.googleapis.com).
-  1. Follow the [setup instructions](/README.md#setup).
-  1. Make changes to the sample, as indicated by `Note: ` comments.
+  1. Follow the [setup instructions](/README.md#setup) to deploy the sample.
+
+This will deploy the base architecture, but the applications will not be functional.
+
+As an example application, follow the instructions in the [RAG Architectures - App Source Code](https://github.com/GoogleCloudPlatform/devrel-demos/tree/main/ai-ml/rag-architectures).
+
+> [!NOTE]
```

Comment:
I wouldn't include this NOTE. This is meant to support the Architecture Center content and AFAIK as long as the Architecture Center content exists, this specific code supporting artifact should be maintained to match.

Reply:
I've clarified that the Example Application is different to the Architecture.